src/tools/mmaps_generator/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ target_link_libraries(mmaps_generator
   g3dlib
   Recast
   Detour
+  shared
```

Comment:
it's strange that every other platform doesn't require this particular link library, could you please elaborate more about this line ?